Dockerfile.ci-rp

```diff
@@ -25,24 +44,25 @@ COPY cmd cmd
 COPY pkg pkg
 COPY test test
 
-# lint
-RUN golangci-lint run --verbose
+# Ensure JS assets are available before generating Go code
+COPY --from=portal-build /build/pkg/portal/assets/v2/ /app/pkg/portal/assets/v2/build
 
-# build
+# Lint, generate, build, and test
+RUN golangci-lint run --verbose
 RUN go generate ./...
 RUN go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=${VERSION}" ./cmd/aro
 RUN go test ./test/e2e/... -tags e2e,codec.safe -c -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=${VERSION}" -o e2e.test
 
-# test
+# Additional tests
 RUN ARO_RUN_PKI_TESTS=nope go run gotest.tools/gotestsum@v1.11.0 --format pkgname --junitfile report.xml -- -coverprofile=cover.out ./...
 RUN hack/fips/validate-fips.sh ./aro
 
 ###############################################################################
-# final is our slim image with minimal layers and tools
+# Stage 3: final is our slim image with minimal layers and tools
 ###############################################################################
 FROM ${REGISTRY}/ubi8/ubi-minimal AS final
 RUN microdnf update && microdnf clean all
-COPY --from=builder /app/aro /app/e2e.test /usr/local/bin/
+COPY --from=builder /app/aro /app/e2e.test /usr/local/bin/   
```

Comment:
looks like some extra whitespace got included here?

Reply:
Fix it